src/lib/datepicker/datepicker.spec.ts

```diff
@@ -96,6 +95,20 @@ describe('MdDatepicker', () => {
         expect(document.querySelector('md-dialog-container')).not.toBeNull();
       }));
 
+      it('open in disabled mode should not open the calendar', async(() => {
```

Comment:
Curious why this test is async. It doesn't look like you're using any of async testing methods?